packages/ai-jsx/package.json

```diff
@@ -69,16 +69,6 @@
         "default": "./dist/cjs/batteries/use-tools.cjs"
       }
     },
-    "./batteries/langchain-wrapper": {
```

Comment:
This was a dead reference. One benefit of our script is that dead references will now be caught.